masonry/src/widget/widget.rs

```diff
@@ -105,6 +106,8 @@ pub trait Widget: AsAny {
     /// The layout strategy is strongly inspired by Flutter.
     fn layout(&mut self, ctx: &mut LayoutCtx, bc: &BoxConstraints) -> Size;
 
+    fn compose(&mut self, ctx: &mut ComposeCtx) {}
```

Comment:
Since you add this already as default impl, should I make a PR for all the other methods here too? (After merging this, to avoid merge-conflicts)

Reply:
I'm adding the default implementations as I convert the passes. I'd rather not add them for passes that still need to recurse on children FWIW.